jekyll/_cci2/insights-partnerships.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: classic-docs
+title: "Insights Partnerships"
+short-title: "Insights Partnerships"
+version:
+- Cloud
+---
+
+## Overview
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 2. You can fix by inserting `{: #overview }` to the next line